InteractiveHtmlBom/ecad/schema/genericjsonpcbdata_v1.schema

```diff
@@ -602,6 +605,22 @@
             "properties": {
             },
             "title": "ExtraData"
+        },
+        "FontData": {
+            "type": "object",
+            "additionalProperties": {
```

Comment:
Use patternProperties with `^?$` regex to enforce single char keys and "additionalProperties": false.